.codegen/__init__.py.tmpl

```diff
@@ -153,7 +153,7 @@ class AccountClient:
         :param workspace: The workspace to construct a client for.
         :return: A ``WorkspaceClient`` for the given workspace.
         """
-        config = self._config.copy()
+        config = self._config.deep_copy()
```

Comment:
Bug fix. The generated code was already updated here

https://github.com/databricks/databricks-sdk-py/commit/9af2630672c866610248243eb53898c16c452960#diff-3847ef3d682f5902da2252a8d7b3a0745d15a4b81ef42cd73f73be158622c5d6R812